examples/.interop/next-prisma-starter/src/server/routers/post.test.ts

```diff
@@ -13,10 +13,8 @@ test('add and get post', async () => {
     text: 'hello test',
     title: 'hello test',
   };
-  const post = await caller.mutation('post.add', input);
-  const byId = await caller.query('post.byId', {
-    id: post.id,
-  });
+  const post = await caller['post.add'](input);
+  const byId = await caller['post.byId']({ id: post.id });
```

Comment:
cc @sachinraja 


Shouldn't this be?:

```suggestion
  const post = await caller.post.add(input);
  const byId = await caller.post.byId({ id: post.id });
```

❓

Reply:
Related: #2284

cc @IgnusG

Reply:
Yes but I'm not sure it's worth it to implement. Type perf is already worsened on the interop routers. We'd have to replicate the structure of a nested router, derived from an interop router.

Reply:
Alright, let's take this as a task on the roadmap